Formula/qt5.rb

```diff
@@ -86,6 +87,7 @@ class Qt5 < Formula
   depends_on :mysql => :optional
   depends_on :postgresql => :optional
   depends_on :xcode => :build
+  depends_on "openssl" => :optional
```

Comment:
Not the 0.9.8 branch, which Apple uses as the "base" for its OpenSSL. I say "base" because Apple has kept selectively plucking security fixes off of the upstream 1.0.1 and 1.0.2 branches so by now it's something of a mutant. 0.9.8 is dead upstream.